pysph/cpy/transpiler.py

```diff
@@ -123,7 +123,7 @@ def __eq__(self, other):
 
 
 class Transpiler(object):
-    def __init__(self, backend='cython'):
+    def __init__(self, backend='cython', incl_cluda=True):
```

Comment:
I think we should include cluda all the time so we might as well remove that option.  Do you see a case where we will not want to include the cluda preamble?

Reply:
The GenericScanKernel for both opencl and cuda include CLUDA by default and it adds a prefix to some of the terms to avoid collision with the user's code. By adding it again, we are redefining some macros, opencl works fine but nvcc gives and error

Reply:
No, I mean we're including the cluda preamble when GenericScanKernel already does it by default (see [this](https://github.com/adityapb/pysph/blob/0622d1b37f8f96975aee5917ea1fd099688e594d/pysph/cpy/cuda.py#L95)). So it ends up being added twice and so we accidentally redefine some macros which gives a compilation error with nvcc

Reply:
Got it.